lerobot/common/datasets/v2/batch_convert_dataset_v1_to_v2.py

```diff
@@ -0,0 +1,880 @@
+#!/usr/bin/env python
```

Comment:
What is the difference between `batch_convert_dataset_v1_to_v2.py` and `convert_dataset_v1_to_v2.py`?
Maybe worth adding to the header of `batch_convert_dataset_v1_to_v2.py`

Reply:
Added a comment at the top
```diff
+ This script is for internal use to convert all datasets under the 'lerobot' hub user account to v2.
```